client.js

```diff
@@ -1,40 +1,39 @@
 'use strict';
 
-const {Client} = require('hazelcast-client');
+const {Client, HazelcastJsonValue} = require('hazelcast-client');
+
+async function mapExample(client){
+    const cities = await client.getMap('cities');
+    await cities.put(1, new HazelcastJsonValue(JSON.stringify({ country: "United Kingdom", city: "London", population: 9_540_576})));
+    await cities.put(2, new HazelcastJsonValue(JSON.stringify({ country: "United Kingdom", city: "Manchester", population: 2_770_434})));
+    await cities.put(3, new HazelcastJsonValue(JSON.stringify({ country: "United States", city: "New York", population: 19_223_191})));
+    await cities.put(4, new HazelcastJsonValue(JSON.stringify({ country: "United States", city: "Los Angeles", population: 3_985_520})));
+    await cities.put(5, new HazelcastJsonValue(JSON.stringify({ country: "Turkey", city: "Ankara", population: 5_309_690})));
+    await cities.put(6, new HazelcastJsonValue(JSON.stringify({ country: "Turkey", city: "Istanbul", population: 15_636_243})));
+    await cities.put(7, new HazelcastJsonValue(JSON.stringify({ country: "Brazil", city: "Sao Paulo", population: 22_429_800})));
+    await cities.put(8, new HazelcastJsonValue(JSON.stringify({ country: "Brazil", city: "Rio de Janeiro", population: 13_635_274})));
+
+    const mapSize = await cities.size();
+    console.log(`'cities' map now contains ${mapSize} entries.`);
 
-function createClientConfig() {
-    return {
-        network: {
-            hazelcastCloud: {
-                discoveryToken: 'YOUR_CLUSTER_DISCOVERY_TOKEN'
-            }
-        },
-        clusterName: 'YOUR_CLUSTER_NAME',
-        properties: {
-            'hazelcast.client.cloud.url': 'YOUR_DISCOVERY_URL',
-            'hazelcast.client.statistics.enabled': true,
-            'hazelcast.client.statistics.period.seconds': 1,
-        }
-    }
+    console.log("--------------------");
 }
 
-async function mapExample(map) {
-    console.log("Now, `map` will be filled with random entries.");
+async function sqlExample(hzClient) {
+    await createMappingForCapitals(hzClient);
 
-    let iterationCounter = 0;
-    while (true) {
-        const randomKey = Math.floor(Math.random() * 100000);
-        await map.put('key' + randomKey, 'value' + randomKey);
-        await map.get('key' + randomKey);
-        if (++iterationCounter === 10) {
-            iterationCounter = 0;
-            const size = await map.size();
-            console.log(`map size: ${size}`);
-        }
-    }
+    await clearCapitals(hzClient);
+
+    await populateCapitals(hzClient);
+
+    await selectAllCapitals(hzClient);
+
+    await selectCapitalNames(hzClient);
+
+    process.exit(0);
 }
 
-async function sqlExample(hzClient) {
+async function createMappingForCapitals(client){
     console.log("Creating a mapping...");
     // See: https://docs.hazelcast.com/hazelcast/5.0/sql/mapping-to-maps
```

Comment:
Can you update the version 
```suggestion
    // See: https://docs.hazelcast.com/hazelcast/5.1/sql/mapping-to-maps
```